android-todo-list/architecture-samples-main/app/src/main/java/com/example/android/architecture/blueprints/todoapp/util/StatsIgUtil.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.android.architecture.blueprints.todoapp.util
+
+import android.app.Application
+import com.statsig.androidsdk.DynamicConfig
+import com.statsig.androidsdk.Statsig
+import com.statsig.androidsdk.StatsigUser
+
+
+/**
+ * Object class StatsIgUtil where all Statsig SDK features will be declared and used.
+ */
+object StatsIgUtil {
```

Comment:
This is an invalid formatting of the word Statsig. Please call this class StatsigUtil

Reply:
Okay.